### PR TITLE
:memo: Document OpenSearch service

### LINKED ISSUE #2103: :memo: Document OpenSearch
### What needs to be documented?

As the successor to Elasticsearch, we need to document the OpenSearch service. It should copy a lot of the same content that's covered in [Elasticsearch](https://docs.platform.sh/configuration/services/elasticsearch.html). Plugins and such are mostly the same, though names should be double checked as some things have been renamed.Migrating should involve a dump and import. It's version 1.2 and available on Grid, DG2, and DG3. 

This should include stating on the Elasticsearch page that the Elasticsearch service won't be receiving more upgrades. And likely marking it as deprecated.

### Additional context

[Additional context](https://github.com/orgs/platformsh/projects/3).

### PR DESCRIPTION
**Pull request by CollierCZ**
## Why

Closes #2103

## What's changed

* Added OpenSearch to registry
* Included OpenSearch example relationship (examples for all apps would be helped by the config parsers being updated to include it)
* Added OpenSearch service page
* Added OpenSearch to changelog
* Deprecate an Elasticsearch version
* Added a note to Elasticsearch about new versions not coming

### CODE REVIEW

File: docs/src/configuration/services/opensearch.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+To do so, add a route to `routes.yaml` that has `search:opensearch` as its upstream
+(where `search` is whatever you named the service in `services.yaml`).
+For example:
+
+```yaml {location=".platform/routes.yaml"}
+"https://os.{default}":
+    type: upstream
+    upstream: search:opensearch
+```
+
```
Comment (trolologuy) [inline docs/src/configuration/services/opensearch.md:84]: Maybe we could add a note on [how to add basic auth](https://docs.platform.sh/administration/web/configure-environment.html#http-access-control) for an environment (bot both this and the ES guides)?

**Review by trolologuy — APPROVED** [review]
- [x] The [final built page](https://pr-2118-mufarza-652soceglkw4u.eu-3.platformsh.site/configuration/services/opensearch.html) is working as expected.
- [x] The information is correct.
- [x] The information is comprehensive.
- [x] The information is concise.
- [x] The information is clear.

File: docs/src/other/changelog.md [hunk 2]
```diff
@@ -7,6 +7,11 @@ description: |
 
 {{< description >}}
 
+## 2022
+
+* **January 2022**
```
Comment (trolologuy) [inline docs/src/other/changelog.md:12]: Technically we are in February, but I assume it refers to the date of implementation?

Comment (CollierCZ) [reply docs/src/other/changelog.md:12]: That was my idea :shrug:

**Pull request merged**